docs/command-line-interface/policy-cli/build.mdx

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Don't we want these docs to just live in openpolicyregistry.io (one place), so that we don't have to keep these docs in sync?

Reply:
They are slightly different since they call out APCR specifically.